charts/kyverno/templates/hooks/exceptions-migration-post-upgrade.yaml

```
@@ -0,0 +1,85 @@
+{{- if .Values.crdsMigration.exceptions.enabled -}}
+  {{- if not .Values.templating.enabled -}}
```

```suggestion
{{- if and .Values.crdsMigration.exceptions.enabled (not .Values.templating.enabled) -}}
```

I think this works, and is a little clearer.  I do notice the other hooks use the 2 conditions so fine to keep as you have it for consistency.

---

I modified it.